docs-v2/integrations/all/circle.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Circle
+sidebarTitle: Circle
+---
+
+API configurations: [`circle`](https://nango.dev/providers.yaml)
```

Comment:
something like this
```suggestion
API configurations: [`circle-so`](https://nango.dev/providers.yaml)
```

Reply:
@bodinsamuel ,that makes perfect sense for distinguishing between the two. I have updated this in the latest [commit](https://github.com/NangoHQ/nango/pull/2577/commits/45e36a2368a9bf464331fc72f4c41e1d59884531).

Reply:
thanks 🙏🏻